tex/paper.tex

```diff
@@ -411,41 +373,195 @@ \section*{Description of the Axelrod Python package}\label{sec:description-of-ax
     \label{fig:grudger_test}
 \end{figure}
 
-To date the library has had contributions from 24 contributors from a variety
-of backgrounds. These contributions have been in terms of strategies. One
-strategy is the creation of an undergraduate mathematics student with little
-prior knowledge of programming. Multiple other strategies were written by a 15
-year old secondary school student. Both of these students are authors of this
-paper. As well as these strategy contributions, vital architectural
-improvements to the library itself have also been received. You can see an
-overview of the structure of the source code in Figure~\ref{fig:overview}.
+You can see an overview of the structure of the source code in
+Figure~\ref{fig:overview}. This shows the parallel collection of strategies and
+their tests. Furthermore the underlying engine for the library is a class for
+tournaments which lives in the \texttt{tournament.py} module. This class is
```

Comment:
Is it not? I thought that's what a module was...

From:

https://www.google.co.uk/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=what%20is%20a%20python%20module

> A module is a Python object with arbitrarily named attributes that you can bind and reference. Simply, a module is a file consisting of Python code. A module can define functions, classes and variables. A module can also include runnable code.

Not defending, just asking what a module is :)